ufficio_soci/tests.py

```diff
@@ -2311,10 +2310,12 @@ def test_download_iscritti(self):
         # autorizzazioni al corso aspirante2 e sostenitore2
         Autorizzazione.objects.create(richiedente=aspirante2, firmatario=direttore_corso, concessa=True,
                                       oggetto_tipo=58, oggetto=partecipazione1, progressivo=1,
```

Comment:
Anche qui `oggetto_tipo=58` va rimosso. `oggetto=partecipazione1` e' abbastanza affinche' Django compili il campo `oggetto_tipo` in autonomia.

Sfortunatamente, non vi e' alcuna garanzia che l'ID del tipo di oggetto corrisponda su diverse installazioni di Gaia, quindi e' probabile che questo test fallisca su altri sistemi.